### DISCUSSION TITLE
Flows: trigger event by creating multiple items, only one is added to payload.

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15603] [Archive]
**I created two items via postman:**

<img width="1356" alt="Screen Shot 2022-09-15 at 3 40 33 PM" src="https://user-images.githubusercontent.com/62306630/190504513-143e228a-1b30-4fb4-9145-7cb1adad62d6.png">

**The Trigger's payload contains ONE item:**

![Screen Shot 2022-09-15 at 3 42 11 PM](https://user-images.githubusercontent.com/62306630/190504816-2af03e96-6f3e-488d-b626-62c9c3240db3.png)

**Is this default behavior? Why don't I get the whole array in the payload?**

**Comment by licitdev** [comment]
@erondpowell Yes that is currently the default behaviour. 
Two action events are triggered, one for each item. 
Please check the flow logs. There should be two run records, one for each created item.

## Background 

POST array of items to create -> CreateMany -> CreateOne that triggers the `items.create` hook 

https://github.com/directus/directus/blob/f9c4422fb177ac07a2f83cbeaa808f3c9e3e02a2/api/src/services/items.ts#L272-L275

**Comment by trevisan92** [comment]
Hello, i search on the documentation but can't find anything. I Need ti trigger a flow to send email with all item sended to createMany but likes you exposed this create a single event per item, it's possibile to trigger only once when multiple items Is created with the same request ? Thanks
